content/docs/0.8.1/advanced-resources/deploy/taint-toleration.md

```diff
@@ -3,32 +3,39 @@ title: Taints and Tolerations
 weight: 3
 ---
 
-## Overview
 If users want to create nodes with large storage spaces and/or CPU resources for Longhorn only (to store replica data) and reject other general workloads, they can taint those nodes and add tolerations for Longhorn components. Then Longhorn can be deployed on those nodes.
 
-Notice that the taint tolerations setting for one workload will not prevent it from being scheduled to the nodes that don't contain the corresponding taints. See [Kubernetes's taint and toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) for details.
+Notice that the taint tolerations setting for one workload will not prevent it from being scheduled to the nodes that don't contain the corresponding taints.
+
+For more information about how taints and tolerations work, refer to the [official Kubernetes documentation.](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+
+# Setting up Taints and Tolerations
```

Comment:
I thought we shouldn't use `#` (header 1) more than once in a page?

Reply:
You're right that the highest level of header should only appear once, at the top. But the website theme uses the `title` defined in the front matter of the markdown file as the highest level heading. So the `#` is effectively level 2 and I think we can use it more than once.